src/EnergyPlus/DataHeatBalance.cc

```diff
@@ -111,7 +111,8 @@ namespace DataHeatBalance {
     int MaxSolidWinLayers(0);               // Maximum number of solid layers in a window construction
                                             // ** has to be big enough to hold no matter what window model
                                             //    each window model should validate layers individually
-    int const MaxSpectralDataElements(800); // Maximum number in Spectral Data arrays.
+//    int const MaxSpectralDataElements(800); // Maximum number in Spectral Data arrays.
```

Comment:
i had to move this variable over to the WindowManagerData struct to get to compile.  We can move it back once DataHeatBalance gets converted to Globals